website/releases/release-0.1.0-incubating.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: "Release 0.1.0-incubating"
+sidebar_position: 1
+---
+
+## [Release 0.1.0-incubating](https://github.com/apache/incubator-xtable/releases/tag/0.1.0-incubating) ([docs](https://xtable.apache.org/docs/how-to))
+This is the first official apache release for Apache XTable, an incubating project under the Apache Software Foundation.
+Apache XTable™ (Incubating) is a cross-table converter for table formats that facilitates omni-directional interoperability across data processing systems and query engines.
+Currently, Apache XTable™ supports widely adopted open-source table formats such as Apache Hudi, Apache Iceberg, and Delta Lake.
+
+## Features
+Apache XTable™ (Incubating) provides users with the ability to translate metadata from one table format to another.
+
+Apache XTable™ (Incubating) provides two sync modes, "incremental" and "full." The incremental mode is more lightweight and has better performance, especially on large tables. If there is anything that prevents the incremental mode from working properly, the tool will fall back to the full sync mode.
+
+This sync provides users with the following:
+
+1. Syncing of data files along with their column level statistics and partition metadata
+2. Schema updates in the source are reflected in the target table metadata
```

Comment:
@sagarlakshmipathy Should we link the features page here to avoid duplication ?

Reply:
For the first release there's only features to list and the improvements we have made to the project.

Reply:
@vinishjail97 I think its better we have it in two places. My 2c, I'd think there would be people looking for features in one place, pointing them to release page would not be a good user exp. Then again, we can't add links to the features page from release notes as well.

I'm open to other ideas.

Reply:
okay let's keep it like it this and not overthink too much now.